src/database/init.sql

```diff
@@ -8,13 +8,17 @@ CREATE TABLE IF NOT EXISTS teams (
   wallet_address VARCHAR(42) UNIQUE,
   bucket_addresses TEXT[],
   is_admin BOOLEAN DEFAULT FALSE,
+  disqualified BOOLEAN DEFAULT FALSE,
+  disqualification_reason TEXT,
+  disqualification_date TIMESTAMP WITH TIME ZONE,
   created_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,
   updated_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP
 );
 
 -- Create team indexes
 CREATE INDEX IF NOT EXISTS idx_teams_api_key ON teams(api_key);
 CREATE INDEX IF NOT EXISTS idx_teams_is_admin ON teams(is_admin);
+CREATE INDEX IF NOT EXISTS idx_teams_disqualified ON teams(disqualified);
```

Comment:
again, should this maybe be "active" rather than "disqualified"?

Reply:
implemented here: [bde5669](https://github.com/recallnet/trading-simulator/pull/10/commits/bde566950c23c546ec676e2bca5f3a2aaf82d550)